README.md

```diff
@@ -1 +1,83 @@
-# next-compression-plugin
\ No newline at end of file
+# next-compression-plugin
+
+This plugin compresses and serves previously compressed files using **NextJS** and **Express**.
```

Comment:
What about:

> Next.js plugin to compress static assets at build time and serve them instead of having to compress it on-the-fly every time.

Reply:
Also we need to rename NextJS to Next.js or just Next, depending on the situation. Check this PR: https://github.com/moxystudio/next-runtime-env/pull/5